src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/AppointmentMappingResource.kt

```diff
@@ -106,6 +103,38 @@ class AppointmentMappingResource(private val mappingService: AppointmentMappingS
     appointmentInstanceId: Long,
   ): AppointmentMappingDto = mappingService.getMappingById(appointmentInstanceId)
 
+  @PreAuthorize("hasRole('ROLE_NOMIS_APPOINTMENTS')")
+  @GetMapping("/mapping/appointments/nomis-event-id/{eventId}")
+  @ResponseStatus(HttpStatus.OK)
```

Comment:
default is `OK` anyway

Reply:
will remove next time (this is widespread so was copied)